src/main/java/armeria/TracingFactory.java

```diff
@@ -13,7 +15,7 @@
   static Tracing create(String serviceName) {
     return Tracing.newBuilder()
         .localServiceName(serviceName)
-        .currentTraceContext(MDCCurrentTraceContext.create())
+        .currentTraceContext(RequestContextCurrentTraceContext.INSTANCE)
```

Comment:
good for now except will change once https://github.com/line/armeria/pull/1328 lands!